helm/inteli-api/templates/deployment.yaml

```diff
@@ -80,6 +80,11 @@ spec:
                 configMapKeyRef:
                   name: {{ include "inteli-api.fullname" . }}-config
                   key: logLevel
+            - name: DSCP_API_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "inteli-api.fullname" . }}-config
+                  key: dscpApiHost
```

Comment:
Again, if this is a fully formatted URI this is fine, if it is however just a port then add a section for the `port`

Reply:
added